pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -62,3 +62,17 @@ func InsertInstructions(node *parser.Node, pos int, instructions string) error {
 	node.Children = append(node.Children[:pos], append(newChild.Children, node.Children[pos:]...)...)
 	return nil
 }
+
+// NextValues returns a slice of values from the next nodes following node. This
+// roughly translates to the arguments to the Docker builder instruction
+// represented by node.
+func NextValues(node *parser.Node) []string {
```

Comment:
Shouldn't we depart from the graph terminology and use something more understandable like `Arguments`? I would prefer this to check if the provided node is a Dockerfile command.